src/test/java/org/gridsuite/modification/server/modifications/LineCreationInNodeBreakerTest.java

```diff
@@ -83,8 +83,8 @@ public void testCreateWithErrors() {
         lineCreationInfosJson = mapper.writeValueAsString(lineCreationInfos);
         mockMvc.perform(post(getNetworkModificationUri()).content(lineCreationInfosJson).contentType(MediaType.APPLICATION_JSON))
             .andExpectAll(
-                status().is4xxClientError(),
```

Comment:
Why is this test changing here ?

Reply:
i found  that test fail because it was a 5xx error not 4xx